synapse/cortex.py

```diff
@@ -4198,15 +4203,14 @@ async def eval(self, text, opts=None):
     async def stormlist(self, text, opts=None):
         return [m async for m in self.storm(text, opts=opts)]
 
-    @s_cache.memoizemethod(size=10000)
-    def getStormQuery(self, text, mode='storm'):
-        '''
-        Parse storm query text and return a Query object.
-        '''
-        query = copy.deepcopy(s_parser.parseQuery(text, mode=mode))
+    async def _getStormQuery(self, args):
+        query = copy.deepcopy(await s_coro.forked(s_parser.parseQuery, args[0], mode=args[1]))
```

Comment:
unit test runtime might be improved by using the new forked() capability inside parser.parseQuery so we still have a cross-cortex cache active?

Reply:
The cross-cortex cache does exist in the subprocesses, and works as expected, but it gets split across the two subprocesses run in the test.  In my last commit I moved forked into parser and it ran ~3mins slower, which also caused a timing-sensitive cron test to fail.  Maybe its the extra async steps or Python lru cache vs our FixedCache?  Let's discuss.

Reply:
Updated per discussion